lib/cli/line-processor.js

```diff
@@ -75,13 +75,13 @@ module.exports = class LineProcessor {
       callback(reportMissingArgument('Host'));
       return;
     }
-    const args = utils.split(tokens, ' ', 2);
-    let [scheme, authority] = utils.split(args[0], '://', 1, true);
+    const args = autocomplete.split(tokens, ' ', 2);
```

Comment:
`split` function here has nothing to do with autocomplete whatsoever, it is just used to split up the arguments provided by the user in CLI.